includes/classes/class-swsales-license.php

```diff
@@ -0,0 +1,136 @@
+                <?php } elseif(!sws_license_is_valid()) { ?>
+                    <p class="sitewide_sales_message sitewide_sales_error"><strong><?php _e('Your license is invalid or expired.', 'paid-memberships-pro' );?></strong> <?php _e('Visit the PMPro <a href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fmembership-account%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dmembership-account%26utm_content%3Dkey-not-valid" target="_blank">Membership Account</a> page to confirm that your account is active and to find your license key.', 'paid-memberships-pro' );?></p>
+                <?php } else { ?>													
+                    <p class="sitewide_sales_message sitewide_sales_success"><?php printf(__('<strong>Thank you!</strong> A valid <strong>%s</strong> license key has been used to activate your support license on this site.', 'paid-memberships-pro' ), ucwords($sws_license_check['license']));?></p>
+                <?php } ?>
+            </div>
+
+            <form action="" method="post">
+			<table class="form-table">
+				<tbody>
+					<tr id="sws-settings-key-box">
+						<td>
+							<input type="password" name="sws-license-key" id="sws-license-key" value="<?php echo esc_attr($key);?>" placeholder="<?php _e('Enter license key here...', 'sitewide-sales' );?>" size="40"  />
+							<?php wp_nonce_field( 'sws-key-nonce', 'sws-key-nonce' ); ?>
+							<?php submit_button( __( 'Validate Key', 'sitewide-sales' ), 'primary', 'sws-verify-submit', false ); ?>
+						</td>
+					</tr>
+				</tbody>
+			</table>
+			</form>
+
+            <p>
+				<?php if ( ! sws_license_is_valid() ) { ?>
+					<a class="button button-primary button-hero" href="https://www.paidmembershipspro.com/membership-checkout/?level=20&utm_source=plugin&utm_medium=pmpro-license&utm_campaign=plus-checkout&utm_content=buy-plus" target="_blank"><?php echo esc_html( 'Buy Plus License', 'paid-memberships-pro' ); ?></a>
+					<a class="button button-hero" href="https://www.paidmembershipspro.com/pricing/?utm_source=plugin&utm_medium=pmpro-license&utm_campaign=pricing&utm_content=view-license-options" target="_blank"><?php echo esc_html( 'View Support License Options', 'paid-memberships-pro' ); ?></a>
+				<?php } else { ?>
+					<a class="button button-primary button-hero" href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fmembership-account%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dmembership-account%26utm_content%3Dview-account" target="_blank"><?php echo esc_html( 'Manage My Account', 'paid-memberships-pro' ); ?></a>
+					<a class="button button-hero" href="https://www.paidmembershipspro.com/membership-account/?redirect_to=%2Fnew-topic%2F%3Futm_source%3Dplugin%26utm_medium%3Dpmpro-license%26utm_campaign%3Dsupport%26utm_content%3Dnew-support-ticket" target="_blank"><?php echo esc_html( 'Open Support Ticket', 'paid-memberships-pro' ); ?></a>
```

Comment:
Should be updated to use sitewide-sales text domain and link to Strangers Studios pages

Reply:
These URL's also need to be updated with latest SS site links. @ideadude

Reply:
I'll merge in then update.